settings.gradle.kts

```
@@ -56,6 +56,7 @@ fun shortenProjectName(project: ProjectDescriptor) {
 
 develocity {
     buildScan {
+        publishing.onlyIf { false }
```

How about a middle ground where it's only published from CI? The logs are lacking in gradle so it has helped us at various times to have the build scan in CI.

```suggestion
        publishing.onlyIf { System.getenv().containsKey("CI") }
```

---

It sounds good!